src/PositionDescriptor.sol

```diff
@@ -26,14 +26,27 @@ contract PositionDescriptor is IPositionDescriptor {
     address private constant WBTC = 0x2260FAC5E5542a773Aa44fBCfeDf7C193bc2C599;
 
     address public immutable wrappedNative;
-    string public nativeCurrencyLabel;
+    bytes32 public immutable nativeCurrencyLabelBytes;
```

Comment:
I suggest making this variable private as we already have a getter

```suggestion
    bytes32 private immutable nativeCurrencyLabelBytes;
```

Reply:
nice

Reply:
i like that